CONTRIBUTING.md

```diff
@@ -85,13 +85,37 @@ pytest tests/unit
 
 ### Building & Testing Documentation
 
+#### Livereload w/automatic rebuild
+
 You can build & preview documentation locally using the following steps.
 
 Change into the doc directory:
 ```commandline
 cd doc
 ```
 
+Run the doc build and host a webserver to preview:
+```commandline
+make serve
+```
+
+You can now open [http://localhost:8000](http://localhost:8000) in your browser to preview the doc.
+
+- The `build/html` directory will contain the generated website files.  
+- `sphinx-autobuild` will host a webserver to preview the docs.  
```

Comment:
Is this supposed to be an explanation of how this works, or instructions? If I'm confused be phrasing, less experienced contributors will be too.

If it's an explanation, I would  remove ambiguity by rephrasing it or removing it.

If it's supposed to be  an instruction step, it's broken both before and after re-installing. 

Before:
```console
$ sphinx-autobuild
bash: sphinx-autobuild: command not found
```

After:
```
$ sphinx-autobuild
usage: sphinx-autobuild [-h] [--port PORT] [--host HOST] [--re-ignore RE_IGNORE] [--ignore IGNORE]
                        [--no-initial] [--open-browser] [--delay DELAY] [--watch DIR]
                        [--pre-build COMMAND] [--version]
                        sourcedir outdir [filenames ...]
sphinx-autobuild: error: the following arguments are required: sourcedir, outdir, filenames
```

Reply:
Good point, I tried to clear it up, removing mention of `sphinx-autobuild` since I see the existing docs don't mention sphinx.
https://github.com/cspotcode/arcade/blob/doc-serve-and-incremental-builds/CONTRIBUTING.md#automatic-rebuild-with-live-reload

Please let me know if you have any suggestions.